e2e/components/Button.test.ts

```diff
@@ -22,7 +22,7 @@ test.describe('Button', () => {
       })
     })
 
-    test('axe @avt', async ({page}) => {
+    test('axe @aat', async ({page}) => {
```

Comment:
What does AAT stand for?

Reply:
@colebemis I take it to mean "automated accessibility tests"